source/SpinalHDL/Structuring/clock_domain.rst

```diff
@@ -373,7 +373,8 @@ SpinalHDL checks at compile time that there is no unwanted/unspecified cross clo
      io.dataOut := area_clkB.buf1
    }
 
-An even simpler method is to use the cross clock domain buffer ``BufferCC(input: T, init: T = null, bufferDepth: Int = 2)`` provided by ``spinal.lib._`` to prevent metastability
+In general, just use 2 filp-flops with dest-clock to prevent metastability. you can use  ``BufferCC(input: T, init: T = null, bufferDepth: Int = 2)`` provided by ``spinal.lib._`` instead.  
```

Comment:
mmm, this phrase is a bit confusing. What about something like this:

> In general you can use 2 or more flip-flop driven by the destination clock domain to prevent metastability. The ``BufferCC(input: T, init: T = null, bufferDepth: Int = 2)`` function provided in ``spinal.lib._`` will instantiate the necessary flip-flop (the number of flip flop will depends on the ``bufferDepth``) to mitigate the phenomena.

feel free to change my example frase (like maybe "the phenomena" can be changed in "metastability problem" (my native language is frown upon you if you repeat a word in the same sentence, or in the same chapter.. but in English is ok) I leave it to your judgment.

Reply:
ok，your suggestion is  even better.  i will merge it soon.